docs/sql-ref-syntax-qry-star.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: global
+title: star (*) Clause
+displayTitle: Star (*) Clause
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+A shorthand to name all the referencable columns in the FROM clause or a specific table reference's columns or fields in the FROM clause.
+The star clause clause is most frequently used in the SELECT list.
+Spark also supports its use in function invocation and certain n-ary operations within the SELECT list and WHERE clause.
+
+### Syntax
+
+```
+[ name . ] * [ except_clause ]
+
+except_clause
+   EXCEPT ( { column_name | field_name } [, ...] )
+```
+
+### Parameters
+
+* **name**
+
+  If present limits the columns or fields to be named to those in the specified referencable field, columns, or table.
+
+* **`*`**
+
+  Collects all the referencable columns in the FROM clause or the optionally specified table_name or view_name into a column list.
+  The list of columns is ordered by the order of table_references and the order of columns within each table_reference.
```

Comment:
This seems duplicated with the overall description, shall we remove it?

Reply:
Agreed